text/0000-location.md

```diff
@@ -0,0 +1,417 @@
+|Location Identifier|location_id|A location is referenced using a location_id (key attribute). For GS1 locations, the location_id is a Global Location Number which is part of GS1 specifications.|"0099474000005"|
+|Namespace|location_namespace|This RFC defines a single location_namespace for GS1; a location in the GS1 location_namespace is called a GS1 location. Note that this design supports extending additional location namespaces in the future.|"01"|
+|Owner|owner/org_id|The identifier of the organization responsible for maintaining the location. For GS1 locations, org_id is the Pike organization which claims ownership of the GS1 company prefix (via the "gs1_company_prefixes" field in the Pike records). The GLN will always start with the company prefix. The smart contract will check that the org claiming ownership lists the matching prefix in their Pike org.||
+
+
+## Transactions
+[transactions]: #transactions
+
+In order to add Locations to state, a transaction must be used. The following 
+transactions and their execution rules are designed for use with Hyperledger
+Sawtooth's Sabre smart contract engine running via Hyperledger Transact, which
+covers all currently implemented DLT backends in Grid.
+
+Locations are managed by submitting transactions to Hyperledger Grid, which will
+process them with the Grid Location smart contract. The following transactions 
+are supported:
+- LocationCreate - create a Location and store it in state.
+- LocationUpdate - update (replace) the properties of a Location already in state.
+- LocationDelete - remove a Location from state.
+ 
+_*Disclaimer: LocationDelete is a potentially hazardous operation and needs to 
+be done with care._
+
+
+## Permissions
+[permissions]: #permissions
+
+Creation of GS1 locations is restricted to agents acting on behalf of the 
+organization that is associated with the GS1 company prefix encoded in the GLN. 
+Like Grid Product, an organization must have the GS1 company prefix in the 
+“gs1_company_prefixes” metadata field for its Pike organization. When a 
+location is created, its owning organization is stored with the location in an 
+“owner” field.
+
+Updates to locations are restricted to agents acting on behalf of the 
+organization stored in the location’s owner field. Only property fields can be 
+updated. They are updated by providing a complete and updated list of the 
+properties, and overwrites the entire list of property values. Location_id, 
+location_type, and owner fields are immutable.
+
+Deletion of locations is restricted to agents acting on behalf of the 
+organization in the location’s owner field. A setting will turn off deletion 
+entirely, with the intent that it could be enabled/disabled by a Grid 
+administrator. Disabling delete is useful because external systems may have 
+references to these locations and deleting them could leave dangling references.
+
+
+## GLN Validation
+[gln validation]: #glnvalidation
```

Comment:
Nice to see this RFC being created.  Is the intent to validate the GLN input with GS1?  Wondering if there is a need to validate the input is coming from the correct GS1 company prefix and those who are associated with it, from a GS1 perspective.  I think stated differently, is how do we ensure bad actors are ruled out and not allowed to input false or misleading data if they simply use a valid GS1 company prefix and the check digit format is correct as part of the GLN?

Reply:
@jsmitchell and @visibleOrigins speak to this in [this comment and thread above ](https://github.com/hyperledger/grid-rfcs/pull/20#discussion_r468588850).